test/AutoExpandbleMap/AutoExpandble_map_utest.cpp

```diff
@@ -2,81 +2,53 @@
 
 #include "Map_test.h"
 
+bool is_succeed(const DiscretePoint2D& Cell_coor, MapParams expected_param) {
```

Comment:
The name is not descriptive. Mb test_expansion_by_cell_update? And you can rm test_auto_expand (it seems redundant).